packages/charting/src/components/LineChart/LineChart.base.tsx

```diff
@@ -263,37 +270,30 @@ export class LineChartBase extends React.Component<
             id={`toolTip${this._uniqueCallOutID}`}
           >
             <div className={this._classNames.calloutContentRoot}>
-              <div className={this._classNames.calloutDateTimeContainer}>
+              <div
+                className={this._classNames.calloutDateTimeContainer}
+                style={yValueHoverSubCountsExists ? { marginBottom: '11px' } : {}}
```

Comment:
>{yValueHoverSubCountsExists ? { marginBottom: '11px' } : {} [](start = 22, length = 59)

yValueHoverSubCountsExists &&{ marginBottom: '11px' }

Reply:
I also tried to shorten with the your suggestion but this was what I was getting:

Type 'false | { marginBottom: string }' is not assignable to type 'CSSProperties | undefined'. Type 'false' is not assignable to type 'CSSProperties | undefined'

React prop style was evaluated into boolean when we have yValueHoverSubCountsExists &&{ marginBottom: '11px' } and that's why it wasn't happy about it. The closest I can get to is to pass undefined instead of empty object. i.e. style={yValueHoverSubCountsExists ? { marginBottom: '11px' } : undefined}

Let me know if there's a better approach